go.mod

```diff
@@ -1,14 +1,83 @@
 module github.com/pfnet-research/node-operation-controller
 
-go 1.16
+go 1.19
 
 require (
-	github.com/google/go-cmp v0.5.5
-	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.15.0
-	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.21.4
-	k8s.io/apimachinery v0.21.4
-	k8s.io/client-go v0.21.4
-	sigs.k8s.io/controller-runtime v0.9.7
+	github.com/google/go-cmp v0.5.8
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.19.0
+	github.com/stretchr/testify v1.8.0
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
```

Comment:
v0.24.3 would be safer because v0.25.0 include some API removals.  I think the removal does not affect the controller, though.

Reply:
If we are using deleted APIs, the build should fail. Since it doesn't fail here, I assume it is unaffected.

I don't think we can permanently not upgrade to v0.25.0, how do we determine that it is safe to do so?

Reply:
ok. let's keep this.